index.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Consider adding a _lang_ attribute to the **html** start tag to declare the language of this document.
In this case **lang="en"**

Reply:
Ok, i understood what i have to do.

Reply:
Updated index.html file on html-lesson branch.